raiden/api/rest.py

```diff
@@ -185,6 +185,11 @@ def _serve_webui(self, file='index.html'):
             assert file
             web3 = self.flask_app.config.get('WEB3_ENDPOINT')
             if web3 and 'config.' in file and file.endswith('.json'):
+                if any(h in web3 for h in ('localhost', '127.0.0.1', '::1')) and\
```

Comment:
please, don't use multiline if statements.

Reply:
And what to do about to long line?

Reply:
compile your conditional beforehand:
```python
my_condition = all(
  first_condition,
  second_condition,
)

if my_condition:
  ...
```

Reply:
Ok.. thank you, @konradkonrad , will keep that in mind on future commits, but for now, managed to shrink this line.